site/src/components/steps.tsx

```diff
@@ -28,20 +28,30 @@ const TutorialSteps: React.FC<TutorialStepProps> = (props) => {
             menuTitle
             order
           }
+          parent {
+            ... on File {
+              relativeDirectory
+            }
+          }
         }
       }
     }
   `);
 
+  let tutorialType = location.pathname.substr(0, location.pathname.lastIndexOf("/"))
```

Comment:
Note: if someone happens to create a subdirectory under `/rest/blog` (`/rest/blog/inner/sample.mdx`), an independent `Steps` will be created. This behavior is undefined now anyways, so the current approach SGTM 👍